### ISSUE TITLE
Feature Request:Save/Load Cross Hotbar

### ISSUE THREAD

**Issue by xyzsys** [RulHolos/Bartender#15]
Hi! I am very excited about this plugin. It may already be planned, but I would like to be able to save/load cross hotbars as well as hotbars. Thank you!

**Comment by RulHolos** [comment]
I'm not sure what you mean by that, can you specify please? :o

**Comment by mrchillington** [comment]
> I'm not sure what you mean by that, can you specify please? :o

The cross bars you use with a controller or that you can just enable to always be showing even when using keyboard and mouse for extra hotbars.

**Comment by xyzsys** [comment]
> The cross bars you use with a controller or that you can just enable to always be showing even when using keyboard and mouse for extra hotbars.

Exactly. Cross hotbars are primarily used when playing on a gamepad, but are sometimes used when 10 hotbars are not enough slots for the player.![image](https://github.com/user-attachments/assets/71c78ca3-3798-41d7-8d4d-7fa4c4218f59)

**Comment by RulHolos** [comment]
oh yeahhhh those
I actually don't know how to handle them and since I don't play with them I don't really know how they work
But I'll look into it!
